### ISSUE TITLE
Remove 1 message to 1 section limitation

### ISSUE THREAD

**Issue by SD10** [MessageKit/MessageKit#586]
This was already done in an experimental branch by PR #537.

I'm pretty sure we can safely remove the 1 message to 1 section limitation in version 1.0. We just need to supply a delegate method for users to control the number of sections. Probably need to update the `numberOfMessages(for:at:in)` to take a `section: Int` argument as well.

Of course, we will leave the underlying logic the same so no one notices a difference.

Only way for me to determine if this is possible is to try and implement it after #579.

**Comment by zhongwuzw** [comment]
Just a thought, if we support multiple cells in one section, I think we should move top/bottom label out of the message cell. 😅

**Comment by SD10** [comment]
> Just a thought, if we support multiple cells in one section, I think we should move top/bottom label out of the message cell. 😅

I want to but it's too many changes already. The good part about the refactor is most of the cell stuff is staying the same. If people want cells without labels, they can build their own 😓

**Comment by zhongwuzw** [comment]
Yep, the pros to move out the labels is: it's good for layout and speed up the time of layout.
But, it's good to keep the original.

**Comment by zhongwuzw** [comment]
See #601
